vm/systemSmartContracts/eei.go

```diff
@@ -137,8 +138,29 @@ func (host *vmContext) GetBalance(addr []byte) *big.Int {
 }
 
 // SendGlobalSettingToAll handles sending the information to all the shards
-func (host *vmContext) SendGlobalSettingToAll(_ []byte, _ []byte) {
-	//TODO: implement this
+func (host *vmContext) SendGlobalSettingToAll(_ []byte, input []byte) {
+	outputTransfer := vmcommon.OutputTransfer{
+		Value:    big.NewInt(0),
+		Data:     input,
+		CallType: vmcommon.DirectCall,
+	}
+
+	for i := uint8(0); i < uint8(host.blockChainHook.NumberOfShards()); i++ {
```

Comment:
Recommend to use `uint16` and take the last 2 bytes into account when creating these `outputAccounts`. The loop is currently limited to 256 shards (sufficient for the near future, but not future-proof, as third-party shards might be theoretically added in the future and exceed 256). Using 2 bytes is also consistent with the current sharding mask suffix.

Reply:
we have to change in a lot of places if we make it bigger